libgnucash/engine/gnc-commodity.cpp

```diff
@@ -239,22 +235,14 @@ static QuoteSourceList single_quote_sources =
     { false, SOURCE_SINGLE, NC_("FQ Source", "YH Finance (FinanceAPI)"), "financeapi" },
 };
 
+// Finance::Quote calls these failover sources
```

Comment:
How about "provides" instead of "calls"?

Reply:
Change has been made.